src/Parse/ParseSchema.php

```diff
@@ -348,6 +363,31 @@ public function addField($fieldName = null, $fieldType = 'String')
         return $this;
     }
 
+      /**
+     * Adding an Index to Create / Update a Schema.
+     *
+     * @param string $indexName Name of the index will created on Parse
+     * @param string $index Key / Value to be saved
+     *
+     * @throws \Exception
+     *
+     * @return ParseSchema indexes return self to create index on Parse
+     */
+    public function addIndex($indexName = null, $index = null)
```

Comment:
Not sure why default values of `null` are needed for both of the args. I think those defaults should be removed considering that a null value would cause this to throw an exception.